lib/ukrust/Makefile.uk

```diff
@@ -0,0 +1,64 @@
+$(eval $(call addlib_s,libukrust,$(CONFIG_LIBUKRUST)))
+
+CINCLUDES-$(CONFIG_LIBUKRUST) += -I$(LIBUKRUST_BASE)/ukrust_sys
+
+RUSTCFLAGS-y += --extern ukrust_sys -L $(LIBUKRUST_BUILD)/ --extern alloc \
```

Comment:
Maybe we want to set the rust flags also only when this lib is included. Imagine someone bringing in an own replacement and doing similar stuff.

Reply:
Fair point, I'll update the pull request accordingly.